### PR TITLE
Fixes #24578 - rename blacklist_kernel_modules snippet

### PR DESCRIPTION
**Pull request by lzap**
Blocker for 1.19 release.

### CODE REVIEW

File: db/seeds.d/020-provisioning_templates_list.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def seeded_templates
         { :name => 'kickstart_ifcfg_get_identifier_names', :source => 'snippet/_kickstart_ifcfg_get_identifier_names.erb', :snippet => true },
         { :name => 'kickstart_networking_setup', :source => 'snippet/_kickstart_networking_setup.erb', :snippet => true },
         { :name => 'preseed_networking_setup', :source => 'snippet/_preseed_networking_setup.erb', :snippet => true },
-        { :name => '_blacklist_kernel_modules.erb', :source => 'snippet/_blacklist_kernel_modules.erb', :snippet => true },
+        { :name => 'blacklist_kernel_modules.erb', :source => 'snippet/_blacklist_kernel_modules.erb', :snippet => true },
```
Comment (mmoll) [inline db/seeds.d/020-provisioning_templates_list.rb]: shouldn't the `.erb` go away, too?

**Comment by ekohl** [comment]
This was my bad. We probably need to document this in the releasing guide.

File: db/seeds.d/020-provisioning_templates_list.rb (hunk 1 above)
Comment (lzap) [reply db/seeds.d/020-provisioning_templates_list.rb]: Oh right! My fix is incorrect, thanks!

**Comment by lzap** [comment]
@ekohl this file is a problem and it happened to everyone, me included.

Rebased, nice catch @mmoll !

**Comment by lzap** [comment]
I think @timogoebel is/plans working on a test that will actually prevent from these kinds of errors. The new template refactoring enabled us to cover renderers with better tests.

**Comment by timogoebel** [comment]
Well, the test won‘t cover this exact thing. We‘re just planing on adding the scaffolding and a small test. But we can iterate on that for sure. :-)

**Review by tbrisker — APPROVED** [review]
Thanks @lzap , test failures don't seem related.

**Pull request merged**